src/main/java/uk/gov/ons/ctp/response/casesvc/service/impl/CaseIACServiceImpl.java

```diff
@@ -92,4 +93,14 @@ private void saveNewIacForCase(final Integer casePk, final String iac) {
             .build();
     caseIacAuditRepo.save(audit);
   }
+
+  public void disableAllIACsForCase(Case caze) {
+    caze.getIacAudits()
+        .forEach(
+            caseIacAudit -> {
+              if (!iacClient.disableIAC(caseIacAudit.getIac())) {
+                log.with("caseId", caze.getId()).error("Failed to disable an IAC for case");
```

Comment:
Throw an exception here

Reply:
If one IAC fails to disable we still want to attempt to disable any other remaining. There's not really anything else this service can do at this point other than log the failure.

Reply:
We resolved this as the IAC's are retrieved from a `NOT NULL` database column they are already null checked.